app/helpers/work_packages_filter_helper.rb

```diff
@@ -50,6 +50,15 @@ def project_work_packages_open_version_path(version, options = {})
     project_work_packages_with_query_path(version.project, query, options)
   end
 
+  def project_work_packages_version_path(version, options = {})
+    query = {
+      f: [
+        filter_object("version_id", "=", version.id)
```

Comment:
I think, if the version is shared with other projects, we have to include those projects via the `subproject_id` filter as well